bihstaging.data-commons.org/manifest.json

```diff
@@ -41,7 +41,7 @@
     "sync_from_dbgap": "False",
     "kube_bucket": "kube_bucket.devplanetv1.gen3",
     "logs_bucket": "logs-devplanetv1-gen3",
-    "useryaml_s3path": "s3://cdis-gen3-users/midrc-bih/user.yaml",
+    "useryaml_s3path": "s3://cdis-gen3-users/qa-midrc-bih/user.yaml",
```

Comment:
I had forgotten that we had a qa-bih environment as well. https://github.com/uc-cdis/gitops-qa/blob/c0aee4c/qa-bih.planx-pla.net/manifest.json#L42
I think this could create some confusion. We don't want external users who are authorized in staging to be authorized in QA as well. We should create a new user.yaml...